### ISSUE TITLE
Waiver Check Screen or Dashboard

### ISSUE THREAD

**Issue by arakla** [sc0v/binder#220]
We should have a page to quickly check if someone has signed a waiver and clearly states which wristband they get. Student Activities told me about multiple people who got building wristbands when they should have gotten non-building wristbands.

Ideally, this screen would do the following:
- An ID is tapped
- Using AJAX, it shows the person name and waiver signed (yes/no) clearly
- If yes, then it says the wristband color (Red or Blue)

The use of AJAX could help page reload times and help speed up wristband distribution.
Also, if we are trying prepare binder to have some persistent data, then a LDAP lookup should be forced to update the Student Class and Department.

The way it should check to see if a person is a builder is by seeing if the person currently has a student class, if they do and are not part of only non-building orgs, then they are a builder.

**Comment by sclark** [comment]
What do you think about this workaround?

**Comment by sclark** [comment]
What do you think about this potential workaround?

**Comment by pkoenig10** [comment]
@sclark that mostly looks good. Can you open a PR and we can discuss there?

**Comment by sclark** [comment]
@pkoenig10 yeah. I wanted to wait for our IS team to merge bug fixes and changes into master before opening PR's for new features, but I will soon.

**Comment by arakla** [comment]
@sclark
I'd say this works, the main other parts to this are what is in #240 and having a separate view where waiver and wristband status very clear. And by clear, I mean a random CMU staff member that has been trained in 1min or less can understand without training.

I suggest having another, similar, view designed for a tablet/touchscreen where looking up a user (Swipe, Tap, or type) will load a page that asks to verify a participant's org and then give visual feedback as to whether they have signed a waiver. I had been thinking something like a large green check and some shape with the wristband color. Tell the IS Team to think back to Prof Hs lectures on design, including the Intro to Design slides, mental models, and Gulf of Execution/Gulf of Evaluation.

**Comment by sclark** [comment]
@arakla thank's for the feedback. Since this refers to issue #240 I'm not sure we will have a working version before this Carnival, but I will talk to the IS team, and keep this issue open going forward.
